sync/class.jetpack-sync-actions.php

```diff
@@ -296,23 +296,23 @@ static function maybe_schedule_sync_cron( $schedule, $hook ) {
 		}
 		$schedule = self::sanitize_filtered_sync_cron_schedule( $schedule );
 
+		// stagged the schedule time by some random fraction of the schedule interval
```

Comment:
> Should we/could we add a test to make sure that the scheduling is random?

If we added such a test, we'd just be testing mt_rand (external function) rather than anything internal. I choose to trust that mt_rand is random :)

> Also, as is, I don't think this would update plugins that exist with the old schedule.

Actually, it does check if the hook is already scheduled, and then un-schedules and re-schedules it - see below:

```php
if ( $schedule != wp_get_schedule( $hook ) ) { //...
```